src/options.js

```diff
@@ -1,7 +1,12 @@
 import Handlebars from 'handlebars';
+import yaml from 'js-yaml';
 import { merge } from './utils';
 
 const defaults = {
+  data: {
+    src: ['src/data/**/*.yaml'],
+    parseFn: (contents, path) => yaml.safeLoad(contents)
```

Comment:
`options` can now take a `parseFn` to run over data files. This allows a user to parse with something else, e.g. `JSON` or something.